examples/server/server.cpp

```diff
@@ -1438,10 +1449,28 @@ struct llama_server_context
                 res.stop = true;
                 res.error = false;
                 res.result_json = {
-                        { "idle",       n_idle_slots       },
-                        { "processing", n_processing_slots },
-                        { "slots",      slots_data         }
+                        { "idle",                            n_idle_slots       },
+                        { "processing",                      n_processing_slots },
+                        { "deferred",                        queue_tasks.queue_tasks_deferred.size() },
+
+                        { "n_prompt_tokens_processed_total", n_prompt_tokens_processed_total},
+                        { "n_tokens_predicted_total",        n_tokens_predicted_total},
+
+                        { "n_prompt_tokens_processed",       n_prompt_tokens_processed},
+                        { "t_prompt_processing",             t_prompt_processing},
+                        { "n_tokens_predicted",              n_tokens_predicted},
+                        { "t_tokens_generation",             t_tokens_generation},
+
+                        { "kv_cache_tokens_count",          llama_get_kv_cache_token_count(ctx)},
```

Comment:
AFAIK `llama_get_kv_cache_token_count` is marked as `(slow, use only for debug)` in the documentation. I'd suggest using `slot.cache_tokens.size()` instead

Reply:
@ggerganov could you advice ? should we sum the all slots `slot.cache_tokens.size()` ? we can also just get ride of this metrics for now

Reply:
How often is the `/metrics` endpoint called (roughly)?

The comment about being slow is to prevent overusing this - say, call it for each decoded token. If the endpoint is called once in a few minutes, I think it is OK